_rename.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env sh
```

Comment:
I tried to run this script with these settings:

```sh
This script will rename your theme and its contents. It will setup you project.

Please enter your theme name:
(This is the name that will be showed in the WordPress admin as the theme name.)
Test theme

Please enter your package name:
(This is the name that will be used for translations in all @package fields and the name of your theme folder.)
(Must be lowercase with no special characters and no spaces. You can use '_' or '-' for spaces)
theme_name

Please enter your theme description:
This is some test description of the theme.

Please enter author name:
Infinum

Please enter author email:
info@eightshift.com

----------------------------------------------------

Your details will be:

Theme Name: Test theme
Description: This is some test description of the theme.
Author: Infinum <info@eightshift.com>
Text Domain: theme_name
Package: theme_name
```

And I got a bunch of `sed: RE error: illegal byte sequence` errors in the console. Can you test this with these settings to see if it will produce the same errors?